noisyatom/requirements/base.txt

```diff
@@ -0,0 +1,14 @@
+appdirs==1.4.0
```

Comment:
Please keep to using requirements.txt as the system deploys using this. Also please add details why you want certain libraries.

Reply:
Changed back to as it was look like before, done